tests/e2e/env/setup.sh

```diff
@@ -1,366 +1,520 @@
 #!/usr/bin/env bash
 
-set -e
-
-# Check for script dependencies.
-# Exit if dependencies are not met.
-if ! command -v jq &> /dev/null
-then
-    echo "The script requires jq library to be installed. For more info visit https://stedolan.github.io/jq/download/."
-    exit 1
-fi
-
-. ./tests/e2e/env/shared.sh
+# Import helper functions.
+# shellcheck disable=SC1091
+source "$(pwd)/tests/e2e/env/helpers.sh"
```

Comment:
@nb Yes. The setup script is run via npm and it's called from the project root

Reply:
If I run it locally, do I also need to run it from this folder? Is anything else stopping us to run it from anywhere, except the `pwd`?

Reply:
> If I run it locally, do I also need to run it from this folder?

Running locally, you can clone the repo, do the basic setup & then run `npm run test:e2e-setup`, which will run the script from the project root. This is the expected approach.

>Is anything else stopping us to run it from anywhere, except the pwd?

There are variables defined under shared.sh, for which the path is defined from project root. So that would be a blocker.

Reply:
But it's because we're also using `pwd` there. The whole thing assumes we run it from a specific directory and breaks in a weird way if not run from the root – maybe we should either stop execution if we're not in the expected directory, or make it work when it's run from anywhere.